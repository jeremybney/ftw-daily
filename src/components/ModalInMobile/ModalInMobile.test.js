import React from 'react';
import { renderDeep } from '../../util/test-helpers';
import ModalInMobile from './ModalInMobile';

describe('ModalInMobile', () => {
  it('no modal when window is missing', () => {
    const props = {
      id: 'TestModalInMobile',
      className: 'test-class-from-props',
      isModalOpenOnMobile: false,
      onManageDisableScrolling: v => v,
    };

    const tree = renderDeep(<div>Content</div>);
    expect(tree).toMatchSnapshot();
  });
});
