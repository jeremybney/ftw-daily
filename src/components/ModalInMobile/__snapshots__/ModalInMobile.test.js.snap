// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalInMobile no modal when window is missing 1`] = `
<div>
  Content
</div>
`;
